src/test/java/jenkins/branch/ParameterDefinitionBranchPropertyTest.java

```diff
@@ -116,6 +120,40 @@ public void jobDecorator_Job_implementing_ParameterizedJob() throws Exception {
         assertThat(new ParameterDefinitionBranchPropertyImpl().jobDecorator((Class) ParamJob.class), notNullValue());
     }
 
+    @Issue("JENKINS-61438")
+    // TODO Enable this test case when jenkins.version includes the patch in stapler/stapler#182 
+    @Ignore("Enable this test when stapler/stapler#182 is merged and the jenkins.version will include the patch")
+    @Test
+    public void choiceParameterIssue() throws Exception {
+        try (MockSCMController c = MockSCMController.create()) {
+            c.createRepository("foo");
+            BasicMultiBranchProject prj = r.jenkins.createProject(BasicMultiBranchProject.class, "foo");
+            prj.setCriteria(null);
+            BranchSource source = new BranchSource(new MockSCMSource(c, "foo", new MockSCMDiscoverBranches()));
+            ParameterDefinitionBranchPropertyImpl instance = new ParameterDefinitionBranchPropertyImpl();
+            instance.setParameterDefinitions(Collections.<ParameterDefinition>singletonList(
+                    new ChoiceParameterDefinition("CHOOSE", new String[] { "a", "b" }, "choose one")
+                    ));
+            source.setStrategy(new DefaultBranchPropertyStrategy(new BranchProperty[]{
+                                                                                      instance, new BasicDummyStepBranchProperty()
+            }));
+            prj.getSourcesList().add(source);
+            r.configRoundtrip(prj);
```

Comment:
Is this where the test fails?

Reply:
I do not remember but I suppose yes. It is when it translate the JSON to ChooseParameter where the method signature have Object and no converter has been registered for Object

Reply:
Yes, just curious whether this caused the entire form submission to fail (HTTP 500 or similar), or if a stack trace got dumped on the system log and the bean was just left unconfigured.

Reply:
for what I remember I got the error page with stacktrack with the jenkins picture in evil mode